Data/Interface/ICustomerService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Data.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Services
+{
+    public interface ICustomerService
+    {
+        public void Create(Customer customer);
+
+        public IEnumerable<Customer> GetAll();
+
+        public Customer GetById(long id);
+        public void Update(Customer customer);
+
+        public void Delete(long id);  
+
+
+    }
```

Comment:
Interface já é publica, redundante explicitar nos métodos também
```suggestion
        void Create(Customer customer);
        IEnumerable<Customer> GetAll();
        Customer GetById(long id);
        void Update(Customer customer);
        void Delete(long id);
    }
```